runbotics-orchestrator/src/main/java/com/runbotics/web/rest/ProcessResource.java

```diff
@@ -262,6 +262,22 @@ public ResponseEntity<ProcessDTO> processSetBotSystem(
         );
     }
 
+    @PreAuthorize("@securityService.checkFeatureKeyAccess('" + FeatureKeyConstants.PROCESS_OUTPUT_TYPE_EDIT + "')")
+    @PatchMapping(value = "/processes/{id}/output-type")
+    public ResponseEntity<ProcessDTO> processSetOutputType(
+        @PathVariable(value = "id", required = false) final Long id,
+        @NotNull @RequestBody ProcessDTO processDTO
```

Comment:
imo you should create new DTO to verify fields you want to be included (see is-triggerable and is-attended endpoints)